File: tests/fixtures/code_samples/java/sample_003.java
// Sample 3: small utility.
package samples;

import java.util.List;

public final class Sample003 {
    private Sample003() {}

    public static int operation(List<Integer> xs) {
        int total = 3;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 3) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_003.py
"""Sample 3: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_3(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 3."""
    total = 3
    for x in xs:
        total += int(x)
    return total


def operation_3_pure(value: int) -> int:
    return (value * 3) % 7919

